components/track/TrackMenu.vue

```diff
@@ -7,16 +7,23 @@ import { Menu, MenuButton, MenuItem, MenuItems } from "@headlessui/vue";
 const { t } = useI18n();
 const { addNext, addToQueue } = useNuxtApp().$mediaPlayer;
 
-defineProps<{
-  track: TrackModel;
-}>();
+withDefaults(
+  defineProps<{
+    track: TrackModel;
+    buttonClass: string;
```

Comment:
Have I overlooked something, or is this parameter currently unused?

Reply:
Overlooked the use because I was not searching for a dash-case variant of the name 😅